pom.xml

```diff
@@ -453,6 +453,15 @@
                     </loggers>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemPropertyVariables>
+                        <jenkins.test.timeout>600</jenkins.test.timeout>
```

Comment:
force timeout for tests to 10 min, JUnit annotations and @Classrule do not work in Jenkins tests there is a monitor in the JenkinsRule class that only changes its timeout with this property.